pythran/pythonic/types/numpy_expr.hpp

```diff
@@ -54,7 +54,7 @@ namespace pythonic {
                 typedef decltype(Op()(std::declval<typename std::remove_reference<Args>::type::dtype>()...)) dtype;
 
                 std::tuple<typename std::remove_reference<Args>::type...> args;
-                array<long, value> shape;
+                types::array<long, value> shape;
```

Comment:
so you keep a long here (I think it's a good choice), so why do you want size_t everywhere?